### PR TITLE
Portfolio: setup and mobile first

### PR DESCRIPTION
**Pull request by Mujeeb4582**
Add the linters file setup
Add different required images
Add index.hml file
Add style.css file
Update the readme file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+	
+	<div class="nav-container">
```
Comment (jasonfelice) [inline index.html]: - [ ] Kindly use the `header` tag for navbar. 😄 
_You have used the `header` tag in the wrong place._

File: index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+					</a>
+				</div>
+			</nav>
+		</div>
+	</div>
+
+	<div class="background-img">
+		<div>
+	<div class="header-container">
+			<header>
```
Comment (jasonfelice) [inline index.html]: - [ ] This `header` tag should not exist here.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+				<div class="menu-icon">
+					<a href="#" class="icon-btn">
+						<img src="./images/Icon - Menu.svg" alt="">
+					</a>
+				</div>
+			</nav>
+		</div>
+	</div>
+
+	<div class="background-img">
```
Comment (jasonfelice) [inline index.html:26]: - [ ] According to the project requirements, you have to use `section` to wrap the headline content.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+  <title>Portfolio</title>
+</head>
+<body>
+	
+	<div class="nav-container">
+		<div class="wrapper">
+			<nav>
+				<div class="logo">Welcome</div>
```
Comment (jasonfelice) [inline index.html]: - [ ] Please make sure the logo is a link. Use `href="#"`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+				</div>
+			</header>
+	</div>
+
+	<div class="social-container">
+		<div class="wrapper">
+			<ul>
+				<div class="box"><li><a href="https://twitter.com/Mujeebu93992980">
+					<img src="./images/twitter.svg" alt="">
+				</a></li></div>
```
Comment (jasonfelice) [inline index.html]: - [ ] It is semantically incorrect to directly nest a `div` inside a `ul`. Please make sure only `li`s are directly nested inside the `ul`.

**Review by jasonfelice — CHANGES_REQUESTED** [review]
Hi @Mujeeb4582, 

Good job so far!
Your design looks amazing, but there are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Great design✔️
No linter errors✔️
Good commit messages✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @Mujeeb4582 , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**